src/permission/config-permission-handler.js

```diff
@@ -33,9 +33,9 @@ const UNDEFINED = 'undefined'
  *                           to load it from the path provided in options.path.
  */
 const ConfigPermissionHandler = function (options, config) {
-  this._ruleCache = new RuleCache(options)
```

Comment:
Passing `options` here is incorrect. `RuleCache` actually wants `options.permission.options`, or `this._permissionOptions`.